packages/amplication-server/src/core/workspace/workspace.resolver.ts

```diff
@@ -221,4 +221,13 @@ export class WorkspaceResolver {
   ): Promise<Coupon> {
     return this.workspaceService.redeemCoupon(user, args);
   }
+
+  @Mutation(() => Boolean)
+  async bulkUpdateWorkspaceProjectsAndResourcesLicensed(
```

Comment:
This mutation must be protected by some kind of administrator authentication.

Reply:
I implemented `GqlCronGuard` => [commit](https://github.com/amplication/amplication/pull/7658/commits/39f344595681a241d68394eb5d3b6b869060b7e8)

I'm unsure if it's more secure than what we did with the custom actions. But I can definitely say that it's more consistent with what we have and usually use, which is Graphql with guards.